ts/session/onions/index.ts

```diff
@@ -61,6 +63,8 @@ export class OnionPaths {
       goodPaths = this.onionPaths.filter(x => !x.bad);
     }
 
+    window.inboxStore?.dispatch(updateOnionPaths(goodPaths[0]));
```

Comment:
probably a good idea to check for the length of goodpath and dispatch [] or undefined if length is not > 0